### PR TITLE
Move wip package to pkg/controller

### PR DESCRIPTION
**Pull request by pmorie**
Completes the work started in #486

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

**Comment by duglin** [comment]
CI busted

**Comment by pmorie** [comment]
Yeah, it's still WIP.  Have a number of things let to do.

File: cmd/controller-manager/app/controller_manager.go [hunk 1]
```diff
@@ -283,7 +282,7 @@ func StartControllers(s *options.ControllerManagerServer,
 		serviceCatalogSharedInformers := informerFactory.Servicecatalog().V1alpha1()
 
 		glog.V(5).Info("Creating controller")
-		serviceCatalogController, err := wip.NewController(
+		serviceCatalogController, err := controller.NewController(
 			coreClient,
 			serviceCatalogClientBuilder.ClientOrDie(controllerManagerAgentName).ServicecatalogV1alpha1(),
 			serviceCatalogSharedInformers.Brokers(),
```
Comment (arschles) [inline cmd/controller-manager/app/controller_manager.go]: worth moving this file to `cmd/controller/...`?

**Comment by arschles** [comment]
Ref https://github.com/kubernetes-incubator/service-catalog/issues/482

File: cmd/controller-manager/app/controller_manager.go (hunk 1 above)
Comment (pmorie) [inline cmd/controller-manager/app/controller_manager.go]: Possibly - I have a feeling we might wind up with multiple controllers.

**Comment by duglin** [comment]
LGTM

**Pull request merged**